Applications/Games/League of Legends/Client/script.js

```diff
@@ -99,36 +84,27 @@ new CustomInstallerScript()
             .checksum(sha1)
             .name(fileName(url))
             .get();
-        
-        mkdir(wine.prefixDirectory+"drive_c/LoL_tmp");
-        mkdir(wine.prefixDirectory+"drive_c/LoL");
-        
-        // Create run script
-        /////////////////////////////////////////
-        var client = wine.prefixDirectory+"drive_c/LoL/run.bat";
-        var batContent = "IF EXIST \"C:\\LoL\\LeagueClient.exe\" (\n"+
-                         "start C:\\LoL\\LeagueClient.exe \n"+
-                         ") ELSE ( start C:\\LoL\\lol.launcher.admin.exe )";
-        writeToFile(client, batContent);  
 
+        return {command: setupFile, args: ["--installdir", "C:\\LoL\\", "--mode", "unattended"]};
+    })
+    .category("Games")
+    .wineDistribution("staging")
+    .wineVersion(LATEST_STAGING_VERSION)
+    .preInstall(function(wine, wizard) {
+        wine.windowsVersion("winxp");
+        wine.d3dx9();
+        wine.overrideDLL().set("native, builtin", ["atl120", "msvcp120", "msvcr120", "vcomp120", "msvcp140"]).do();
+        wine.enableCSMT();
 
-        // Extract the msi file and create a link to it
-        ///////////////////////////////////////////////
-        wine.run(setupFile, ["/extract:C:\\LoL_tmp", "/exenoui"]).wait();
-        var regionMSIPath = wine.prefixDirectory+"drive_c/LoL_tmp/LoL."+regionID+".msi";
-        var linkPath = wine.prefixDirectory+"drive_c/LoL_tmp/LoL.msi";
-        lns(regionMSIPath, linkPath);
+        mkdir(wine.prefixDirectory + "drive_c/LoL");
 
-        wine.d3dx9();
-        wine.vcrun2015();
+        // Create run script to start the right exe
+        /////////////////////////////////////////
+        var client = wine.prefixDirectory + "drive_c/LoL/run.bat";
+        var batContent = "start C:\\LoL\\LeagueClient.exe";
+        writeToFile(client, batContent); 
     })
     .postInstall(function(wine, wizard) {
```

Comment:
If `postInstall` contains no more content, I think you can try to safely remove it :)